core/src/main/java/org/mule/api/security/tls/TlsConfiguration.java

```diff
@@ -112,7 +126,8 @@
 {
     public static final String DEFAULT_KEYSTORE = ".keystore";
     public static final String DEFAULT_KEYSTORE_TYPE = KeyStore.getDefaultType();
-    public static final String DEFAULT_KEYMANAGER_ALGORITHM = KeyManagerFactory.getDefaultAlgorithm();
+    // Don't use default as in Java 7 it's SunX509 or implementation dependant, starting from Java 8 it's PKIX
```

Comment:
Are you sure about this? While doing some debugging recently with Java 8 I kept getting SunX509 as default.